site.conf

```diff
@@ -53,6 +53,7 @@
 		enabled = true,
 		fastd = {
 		         methods = {'salsa2012+umac'},
```

Comment:
muss hier nicht noch null oder "null" dazu?

Reply:
In dem Abschnitt der Doku geht es um die Gateways/fastd Server.
Laut Kommentar von NeoRaider zum Performance Mode Commit wird in der site.conf nur das configurable-Flag gesetzt und auf dem fastd Server "null" erlaubt.
"null" wird nur am Anfang der Method List ergänzt, wenn der Performance Mode Standard sein soll: https://github.com/freifunk-gluon/gluon/pull/362

Ich kann das leider nicht testen, da "null" noch nicht auf den fastd Servern/Gateways von Freifunk Flensburg erlaubt wurde.
"null" ist übrigens die Methode und "none" die dazugehörige Cipher. In der site.conf bzw. der fastd Config gibt man die Methoden an, also "null".